docs/installingLocal.md

```diff
@@ -28,16 +28,16 @@ This page describes the procedure to run PrairieLearn within Docker, but using a
   # Repeat after switching branches, pulling new code, or editing Python dependencies in `plbase` image.
   # If editing code in `packages/`, you should also repeat either this command or `make build`.
   make deps
-
+  
```

Comment:
Huh... our inline JS formatter doesn't add spaces in this situation, but the SQL formatter does. Weird!

Reply:
Yeah I am not a fan of the spaces it adds either.

Reply:
Weird to me either. 🤣 Embedded in markdown seems not been taken into account in the plugin currently.